src/apprunner/explorer/apprunnerServiceNode.ts

```diff
@@ -116,16 +117,27 @@ export class AppRunnerServiceNode extends CloudWatchLogsBase implements AWSResou
     }
 
     private async updateOperation(): Promise<void> {
-        const operations = (await this.client.listOperations({ MaxResults: 1, ServiceArn: this._info.ServiceArn }))
-            .OperationSummaryList
-        const operation = operations && operations[0]?.EndedAt === undefined ? operations[0] : undefined
-        if (operation !== undefined) {
-            this.setOperation(this._info, operation.Id!, operation.Type as any)
-        }
+        return this.client
+            .listOperations({ MaxResults: 1, ServiceArn: this._info.ServiceArn })
+            .then(resp => {
+                const operations = resp.OperationSummaryList
+                const operation = operations && operations[0]?.EndedAt === undefined ? operations[0] : undefined
+                if (operation !== undefined) {
+                    this.setOperation(this._info, operation.Id!, operation.Type as any)
+                }
+            })
+            .catch(err => {
+                // Apparently App Runner can rarely list deleted services with the wrong status
+                getLogger().warn(
+                    `Unable to list operations for service "${this._info.ServiceName}", it will not be shown.`
```

Comment:
nit: unnecessary entroy that "Unable to" is used here, but "Failed to" is used on the next line.

Reply:
The wording was wrong in general. If we can't get the operation, the service still shows but can't pause/run/deploy